hw/ip/adc_ctrl/doc/_index.md

```diff
@@ -81,22 +81,23 @@ Note that the time taken in this step depends on the properties of the ADC.
 
 In active operation the controller is in continuous scanning mode:
 * The ADC is continually powered on.
-* The sampling cycle time is the time taken for the ADC to take two samples (450us) plus NNN (fill in based on implementation but probably 4-10)) number of slow clock cycles x (5us; always-on clock frequency is 200KHz).
-* The debounce timer will trigger the filter status (and interrupt) after {{< regref "adc_sample_ctl" >}} multiplied by the continuous sampling cycle time.
+* The sampling cycle time is the time taken for the ADC to take two samples (450us) plus processing time from the adc controller.
+* The debounce timer will trigger the {{< regref "filter_status" >}} and interrupt after a configurable number of matching ADC samples have been seen, as determined by {{< regref "adc_sample_ctl" >}}.
 
 For low power operation the periodic scanning mode can be used.
 In this mode samples are taken using a slower periodic sampling cycle time with the ADC powered down most of the time.
 Once a small number of cycles have hit the filter with periodic scanning then the controller switches to continuous scanning in order to more accurately debounce the signal.
 In low power mode:
-* The ADC is only powered up for {{< regref "adc_pd_ctl.pwrup_time" >}} slow clock cycles plus the time taken for the ADC to take two samples plus NNN slow clock cycles.
-* The total time between samples for a periodic scan is the time the ADC is powered up plus {{< regref "adc_pd_ctl.wakeup_time" >}} cycles of the slow clock.
-* The time taken to transition to the continuous scanning mode is the total time of a slow scan multiplied by the {{< regref "adc_lp_sample_ctl" >}}.
-* The debounce timer will trigger the filter status (and interrupt) after the time taken to transition to fast mode plus {{< regref "adc_sample_ctl" >}} multiplied by the fast sampling cycle time.
+* The ADC is periodically powered up to take samples; this interval is determiend by {{< regref "adc_pd_ctl.wakeup_time" >}}.
+* Similar to normal operaiton, the ADC power-up delay is controlled by {{< regref "adc_pd_ctl.pwrup_time" >}}.
+* Once the ADC is powered up, two samples are taken and compared to the filter thresholds.
+* If a configureable number of matches, as determined by {{< regref "adc_lp_sample_ctl" >}}, are seen, the adc controller transitions to normal operation for continuous sampling.
```

Comment:
nit: `configureable -> configurable`

Reply:
thanks for noticing. I'll try to remember turning on spell check in the future...